### PR TITLE
⚠️ CONFLICT! Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request: CONFLICT #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-ansible-role.git`](https://github.com/cisagov/skeleton-ansible-role.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

The `lineage/skeleton` branch has **one or more unresolved merge conflicts**
that you must resolve before merging this pull request!

## How to resolve the conflicts ##

1. Take ownership of this pull request by removing any other assignees.

1. Clone the repository locally, and reapply the merge:

    ```console
    git clone git@github.com:cisagov/ansible-role-xfce-cool.git ansible-role-xfce-cool
    cd ansible-role-xfce-cool
    git remote add skeleton https://github.com/cisagov/skeleton-ansible-role.git
    git remote set-url --push skeleton no_push
    git switch develop
    git checkout -b lineage/skeleton --track origin/develop
    git pull skeleton HEAD
    git status
    ```

1. Review the changes displayed by the `status` command.  Fix any conflicts and
   possibly incorrect auto-merges.

1. After resolving each of the conflicts, `add` your changes to the
   branch, `commit`, and `push` your changes:

    ```console
    git add .github/workflows/build.yml 
    git commit
    git push --force --set-upstream origin lineage/skeleton
    ```

    Note that you may _append_ to the default merge commit message
    that git creates for you, but *please do not delete the existing
    content*.  It provides useful information about the merge that is
    being performed.

1. Wait for all the automated tests to pass.

1. Check the "Everything is cool" checkbox below:

    - [x] ✌️ The conflicts in this pull request have been resolved.

1. Mark this draft pull request "Ready for review".

------------

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

**Review by dav3r — APPROVED** [review]

**Review by jsf9k — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -33,8 +33,10 @@ None.
 
 ## Role Variables ##
 
-- `jetbrainsmono_version` - the version of the JetBrainsMono font to
-  download from JetBrains/JetBrainsMono.  Defaults to "2.241".
+| Variable | Description | Default | Required |
+|----------|-------------|---------|----------|
+| jetbrainsmono_version | The version of the JetBrainsMono font to
+  download from JetBrains/JetBrainsMono. | `2.241` | No |
```
Comment (mcdonnnj) [inline README.md:39]: Breaking the line causes a failure in rendering:

image


```suggestion
| jetbrainsmono_version | The version of the JetBrainsMono font to download from JetBrains/JetBrainsMono. | `2.241` | No |
```

**Review by mcdonnnj — CHANGES_REQUESTED** [review]
Please update the image used here
https://github.com/cisagov/ansible-role-xfce-cool/blob/13e99077f6be4eb8e2915d3169a873ef8627c535/molecule/debian/molecule.yml#L25-L26
and add the Fedora 34 configuration to `molecule/redhat/molecule.yml`.

File: README.md (hunk 1 above)
Comment (jsf9k) [reply README.md]: Good catch!

**Comment by dav3r** [comment]
> Please update the image used here
> https://github.com/cisagov/ansible-role-xfce-cool/blob/13e99077f6be4eb8e2915d3169a873ef8627c535/molecule/debian/molecule.yml#L25-L26
> 
> 
> and add the Fedora 34 configuration to `molecule/redhat/molecule.yml`.

Excellent catches @mcdonnnj!  Resolved in https://github.com/cisagov/ansible-role-xfce-cool/pull/21/commits/f1ea1a172086b67d9d6a2171eb7a2ede5bf863a0 and https://github.com/cisagov/ansible-role-xfce-cool/pull/21/commits/a6fb6a8420e27a0676997dd57f2249e82d64b2a4.

**Review by mcdonnnj — APPROVED** [review]
Thanks for fixing those configuration! :shipit:

**Pull request merged**